tests/test_cmake_build/test.py

```diff
@@ -3,5 +3,10 @@
 
 import test_cmake_build
 
+from .. import env
+
+if env.PY3:
```

Comment:
Without the matching guard in C++ we are as much creating a problem as we are solving one.